.env.temp

```diff
@@ -0,0 +1 @@
+PUSHER_DEV_SUFFIX=-14fec3ac47964662914a7b6a69ba5e41
```

Comment:
It looks like you committed this file on accident. If so, please put up a PR to remove it. If not, what's it for?

Reply:
lolllllllllll it's still there 😅

Reply:
oh shoot never saw @neil-marcellini's original message. I'm pretty sure I committed it on accident but I'll double check rq and make a pr to remove it.

Reply:
I think you should keep it and see how long it lasts :trollface: